.drone.yml

```diff
@@ -30,16 +44,49 @@ pipeline:
     when:
       event: pull_request
 
+  build_acceptance:
+    image: docker:1.11
+    environment:
+      - DOCKER_HOST=tcp://127.0.0.1:2375
+    commands:
+      - docker build -f Dockerfile-acceptance -t acceptance .
+    when:
+      branch: master
+      event: push
+
   test_acceptance:
     image: docker:1.11
     environment:
       - DOCKER_HOST=tcp://127.0.0.1:2375
     commands:
-      - docker run -e NODE_ENV=ci -d --net host ukvi-complaints
+      - docker run -e NODE_ENV=ci -d --net host app
       - docker run --rm --net host acceptance
     when:
       event: pull_request
 
+  test_acceptance:
+    image: docker:1.11
+    environment:
+      - DOCKER_HOST=tcp://127.0.0.1:2375
+    commands:
+      - docker run -e NODE_ENV=ci -d --net host app
+      - docker run --rm --net host acceptance
+    when:
+      branch: master
+      event: push
+
+  image_to_quay:
+    image: docker:1.11
+    environment:
+      - DOCKER_HOST=tcp://127.0.0.1:2375
+    commands:
+      - docker login -u="ukhomeofficedigital+ukvi_complaints" -p=${DOCKER_PASSWORD} quay.io
```

Comment:
we are passing in the docker password - would it make sense to also pass in the app name? That way the same file could be shared across applications